### ISSUE TITLE
Additional end user functionality

### ISSUE THREAD

**Issue by tliutony** [KordingLab/ASKLB#19]
- [x] ability to select final model with final test accuracy
- [x] show final model true AUC
- [ ] ability to download final predictions
- [x] clamp thresholdout output to be max 1
- [x] truncation of sig figs for readability
- [x] Azure configuration
- [x] specify test set as a fraction of the training data
